### ISSUE TITLE
Add per-tensor quantization option into HW configs

### ISSUE THREAD

**Issue by AlexKoff88** [openvinotoolkit/nncf#366]
In some cases, a user should have the ability to select per-tensor quantization for both weight and activations by defining the scheme explicitly in the NNCF quantization config. So the agreement is to add these options for 8-bit quantization to CPU, GPU and VPU config (options for 2 and 4 bits should be kept as is).

**Important note**: the presets described in #361 should select per-channel quantization by default whenever it is possible.

**Comment by AlexKoff88** [comment]
New part of CPU HW config:

```
"target_device": "CPU",
"config": {
    "quantization": {
        "q8_a": {
            "bits": 8,
            "mode": [
                "symmetric",
                "asymmetric"
            ],
            "granularity": "pertensor"
        },
        "q8_a_ch": {
            "bits": 8,
            "mode": [
                "symmetric",
                "asymmetric"
            ],
            "granularity": ["perchannel", "pertensor"]
        },
        "q8_w_sym": {
            "bits": 8,
            "mode": "symmetric",
            "level_low": -128,
            "level_high": 127,
            "granularity": ["perchannel", "pertensor"]
        },
        "q8_w_asym": {
            "bits": 8,
            "mode": "asymmetric",
            "granularity": ["perchannel", "pertensor"]
        }
    }
}
```

VPU:

```
"target_device": "VPU",
"config": {
    "quantization": {
        "q8_tn": {
            "bits": 8,
            "mode": "asymmetric",
            "granularity": "pertensor"
        },
        "q8_ch": {
            "bits": 8,
            "mode": "symmetric",
            "granularity": ["perchannel", "pertensor"]
        },
        "q4_tn": {
            "bits": 4,
            "mode": "symmetric",
            "granularity": "pertensor"
        },
        "q4_ch": {
            "bits": 4,
            "mode": "symmetric",
            "granularity": "perchannel"
        },
        "q2_ch": {
            "bits": 2,
            "mode": "symmetric",
            "granularity": "perchannel"
        }
    }
}
```

**Comment by vshampor** [comment]
@AlexKoff88 please open a PR to adjust the HW config itself and I'll handle the rest.

**Comment by vshampor** [comment]
Closed via #443
